task_2.py

```diff
@@ -0,0 +1,44 @@
+from fractions import Fraction
+
+
+def CheckFraction(param):
+    while True:
+        str = input(f"{param} ")
+        if str.count("/") == 1:
+            list = str.split("/")
+            try:
+                a = int(list[0])
+                b = int(list[1])
+                return (a, b)
```

Comment:
Можно сразу помещать в переменные
```suggestion
            list = str.split("/")
            try:
                a, b = map(int, str.split("/"))
                return a, b
```
 и дальше также, при возврате, распаковать в две переменные, а не кортежем и индексацию не нужно использовать